### ISSUE TITLE
Mismatch Between uniffi_contract_version and bindings_contract_version in uniffi 0.26

### ISSUE THREAD

**Issue by Catalpam** [antoniusnaumann/cargo-swift#49]
Description:
I've encountered a version mismatch issue between the uniffi_contract_version generated by the uniffi 0.26 library and the bindings_contract_version generated by cargo swift. This mismatch leads to compatibility problems and runtime errors when attempting to use generated bindings with the Rust library.

**Comment by antoniusnaumann** [comment]
The latest supported UniFFI version for now is v0.25.3

I will update this in the next few days.

**Comment by antoniusnaumann** [comment]
I updated to UniFFI 0.26, you can try it out by installing from git:

```sh
cargo install -f --git https://github.com/antoniusnaumann/cargo-swift.git
```

Let me know if you run into any issues.

**Comment by antoniusnaumann** [comment]
This should be solved in the 0.6.0 version published on crates.io
